### PR TITLE
Feat/menu component

### PR DESCRIPTION
**Pull request by dima716**
- сreate new shared menu component
- add no-tick option for dropdown-list item
- create new demo-shared component for demostrating shared components

### CODE REVIEW

**Comment by dima716** [comment]
@yuriMalakhov There are some troubles with protractor. Can you have a look at [travis console](https://travis-ci.org/aitarget/aitarget-components/builds/251933697?utm_source=github_status&utm_medium=notification)?

File: src/lib/shared/shared.module.ts [hunk 1]
```diff
@@ -34,7 +39,10 @@ import { TickComponent } from './components/tick.component';
     TickComponent,
     ValuesPipe,
     PluralPipe,
-    ValidateMessageComponent
+    ValidateMessageComponent,
+    FbaMenuComponent,
+    ArrowDropComponent,
+    PreloaderDotsComponent
```
Comment (yuriMalakhov) [inline src/lib/shared/shared.module.ts]: Из-за этого не работают тесты.

Comment (yuriMalakhov) [inline src/lib/shared/shared.module.ts]: Beware of $PASS and $EMAIL environment variables needed to run e2e tests. Use our test user credentials or your own.

**Comment by yuriMalakhov** [comment]
@dima716 Please, squash your 8 commits to up to 3 commits, so the history becomes cleaner.

**Comment by dima716** [comment]
@yuriMalakhov squashed into 2 commits. It seemed more logical to leave only two.

**Comment by yuriMalakhov** [comment]
Let's squash again. Unnecessary empty commit https://github.com/aitarget/aitarget-components/pull/188/commits/ff8c8e90b42fb22717ebdd2e40c74fe31d8ca4ff

**Pull request merged**